src/index.css

```diff
@@ -0,0 +1,68 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;600;700;800&display=swap');
+
+body {
+  margin: 0;
+  font-family: 'Montserrat', sans-serif;
+  background: #f5f6fa;
+}
+
+.wrapper {
+  padding: 4.3rem 3.75rem 7.688rem 6.25rem;
+}
+
+.panel-bg {
+  background: #fafafa;
+  border-radius: 4px;
+
+  /* height: 68.563rem; */
+  box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.4);
+  padding: 0 0 9.25rem;
+}
+
+.navbar {
+  background: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 3rem;
+}
+
+.menu-nav {
+  display: flex;
+  align-items: center;
+}
+
+.menu-nav h1 {
+  font-size: 1.9rem;
+  font-weight: 800;
+  color: #0290ff;
+}
+
+.menu {
+  list-style-type: none;
+  display: flex;
+  gap: 2rem;
+}
+
+.menu li a {
+  text-decoration: none;
+  text-transform: uppercase;
+  font-size: 0.958rem;
+  font-weight: 500;
+  letter-spacing: 1.9px;
+  color: #121212;
+
+  /* opacity: 0.5; */
```

Comment:
- [ ] [OPTIONAL] Same here kindly make sure there is no commented code in your codebase